src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -25,7 +25,7 @@ public function assertResponseOk()
 	 */
 	public function assertResponseStatus($code)
 	{
-		return PHPUnit::assertEquals($code, $this->response->getStatusCode());
+		return PHPUnit::assertEquals($code, $this->response->getStatusCode(), 'Expected status code '.$code.', got '.$actual);
```

Comment:
@shehi That would been an issue anyway. PHP would have behaved the same when you try to concatenate them.